controllers/app.controllers.js

```diff
@@ -47,3 +48,17 @@ exports.getCommentsByReviewId = (req, res, next) => {
       next(err);
     });
 };
+
+exports.postCommentByReviewId = (req, res, next) => {
+  const reviewId = req.params.review_id;
+  Promise.all([
+    checkReviewIdExists(reviewId),
```

Comment:
It's not necessary to check if the review exists in this case. If it doesn't exist it will be caught with an SQL error. 

The only reason you need to check if it exists is when you need to differentiate between review doesn't exist and when the review exists but has no comments, both of which returned an empty array.